packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -329,17 +343,24 @@ export default class ImageEdit implements EditorPlugin {
      * Create editing wrapper for the image
      */
     private createWrapper(operation: ImageEditOperation | CompatibleImageEditOperation) {
-        // Wrap the image with an entity so that we can easily retrieve it later
+        //Clone the image and insert the clone in a entity
+        const clone = this.image.cloneNode();
+        const wrappedImage = wrap(clone, KnownCreateElementDataIndex.ImageEditWrapper);
+
+        this.clonedImage = wrappedImage.firstElementChild as HTMLImageElement;
+
+        // Wrap the cloned image with an entity so that we can easily retrieve it later
         const { wrapper } = insertEntity(
```

Comment:
Since the editing wrapper is not part of editor now, I don't think we still need to "insertEntity"